tests/scripts/test_lib

```diff
@@ -750,6 +763,10 @@ function fio_common_cmdline()
 
 	fi
 
+	if [ "$(devtype ${dev})" == "smr" ]; then
+		fiocmd+=" --zonemode=zbd"
```

Comment:
Doing this only is far from enough because the drive may actually have absolutely nothing to read if all zones are empty and we have no conventional zones. If the drive has conventional zones, then read tests will endup reading only the tiny conventional zones LBA range which will not generate enough seek overhead to get a correct results for most tests that rely on failed IOs due to the IO time limits being exceeded. So we need preconditioning of the SMR drive by filling zones before starting the tests. E.g. filling up 1 zone every 100 zone should be enough.